modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2990,7 +3008,7 @@ export class Wallet implements IWallet {
     }
 
     // ECDSA TSS uses TxRequestFull
-    if (this.baseCoin.getMPCAlgorithm() === 'ecdsa' || params.apiVersion === 'full' || this._wallet.type === 'cold') {
+    if (!this.tssUtils?.supportedTxRequestVersions().includes('lite') || params.apiVersion === 'full') {
```

Comment:
Can we just define the apiVersion we are using at the top of this fn. would be easier to read/reuse.